### ISSUE TITLE
Context menu not keyboard accessible

### ISSUE THREAD

**Issue by anneliev** [sebgroup/green#1152]
### Bug already reported?

- [X] I confirm that I have checked if the bug already has been reported

### For which framework/library you are reporting the bug

Angular

### Component name

More menu

### Description

When navigation with the keyboard to the More menu and press enter, next focus is not i the opened menu. When you use tab, focus moves to the top of the browser (in chrome), through adress bar, to the opened menu and back to the More button. You cannot move focus to the opened menu with arrow keys either.

### Steps To Reproduce

Upgrade to version @sebgroup/green-angular": "^2.1.2"
Run application
Navigate using the keyboard to a More menu button
Press enter
Try to navigate to the items in the opened context menu

### Current Behaviour

It is not possible to select the items in the opened context menu using the keyboard

### Expected Behaviour

It should be possible to navigate using only the keyboard

**Comment by splashdust** [comment]
Hi @anneliev, thank you for reporting this issue!

I'm not able to reproduce it when testing it in Storybook: https://sebgroup.github.io/green/latest/angular/?path=/docs/components-context-menu--custom-template-default

If possible, it would be great if you could send me a link on Teams to where the issue can be reproduced. 

For reference, here is how the keyboard interactions are supposed to work:

* Enter or key-down when focus is on trigger opens the popover and focus moves to the first option.
* Arrow-up and arrow-down to move focus between options while the popover is open.
* Tab should close the popover and return focus to the trigger button.

Basically, it should follow the `menuitem` role recommendations from MDN: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/menuitem_role

**Comment by anneliev** [comment]
Hi,
As you say it works as intended in Storybook, but not when we implement it. You can see it here : https://internalapps.tst.sebank.se/nexus/payments/payment-investigation

Regards, Annelie

**Comment by splashdust** [comment]
> Hi, As you say it works as intended in Storybook, but not when we implement it. You can see it here : 
> 
> Regards, Annelie

Thank you, I will take a look!

**Comment by splashdust** [comment]
@anneliev It seems like this issue originates somewhere else in the app (or shell). Probably something like focus or keyboard event getting intercepted before it reaches the component, and therefore the component can't respond to it.

I will close this issue now, but feel free to re-open, or open a new issue, if you find that there is something wrong with the component itself.
